report/distributionList.go

```diff
@@ -0,0 +1,52 @@
+package report
+
+import (
+	"io/ioutil"
+
+	"github.com/davecgh/go-spew/spew"
+	"gopkg.in/yaml.v2"
+)
+
+// DistributionFormat is for unmarshiling a email distributionList file
+type DistributionFormat struct {
+	Severity SeverityDistribution `yaml:"severity"`
+}
+
+// SeverityDistribution list of emails ordered by severity
+type SeverityDistribution struct {
+	Debug []string `yaml:"debug,omitempty"`
+	Info  []string `yaml:"info,omitempty"`
+	Warn  []string `yaml:"warn,omitempty"`
+	Error []string `yaml:"error,omitempty"`
+	Fatal []string `yaml:"fatal,omitempty"`
+}
+
```

Comment:
I tried something similar in gocd.go with PipelineConfig struct holding an inline Pipeline struct. But then I couldn't use the Pipeline struct as arguments or return objects, it was really limiting. Same situation in the sqlhealthcheck.go with the Format &  SQLHealthCheck structs. If we want to use SeverityDistribution outside of the DistributionFormat, it needs to be explicitly stated as a typed struct. I'll look into the GoDocs if there is a way around that.